starknet_devnet/blueprints/rpc/structures/payloads.py

```diff
@@ -226,6 +226,12 @@ class RpcDeclareTransaction(RpcTransactionCommon):
     sender_address: Address
 
 
+class RpcDeclareV2Transaction(RpcDeclareTransaction):
```

Comment:
Should these types be framed in the same way we already have it with broadcasted types (notice the V1, V2 and Union):
https://github.com/0xSpaceShard/starknet-devnet/blob/4a8d028c5babea9616041781d21e337b55d7936d/starknet_devnet/blueprints/rpc/structures/payloads.py#L155-L170